src/app/[locale]/specs/account/page.mdx

```diff
@@ -0,0 +1,90 @@
+
+The general expectation is that downstream services will not list an account as `active` if it is inactive at an upstream service. However, the concept of “upstream” is informal, and services may need to define their own policies for some corner-cases. For example, an AppView might consume from multiple independent relay providers, who report differing statuses for the same account, due to policy differences. If the network connection between a relay and PDS is disruptive, they may report different statuses for the same account. 
+
+If there is doubt about about an account’s general status in the network, the account’s current active PDS host can be queried using the `com.atproto.sync.getRepoStatus` endpoint. If the account is not `active` there, it is generally expected to not be active elsewhere, especially in the case of user-initiated actions (deactivation or deletion).
+
+### Content Deletion
+
+Separate from overall account status, individual pieces of content may be removed. When accounts remove records from their public repositories, hosting services should remove public access to that content.
+
+Applications which would benefit from explicit “tombstones” (indications that content previously existed) should explicitly design them in to Lexicon schemas. In the absence of explicit application-level tombstones, services are expected not to differentiate between content which has never existed and content which has been entirely deleted. Note that this is different from acknowledging account-level deletion, which is encouraged.
+
+## PDS Account Migration
+
+At a high level, the current PDS host for an account is indicated in the account’s DID document. If a DID document is updated to point at a new PDS host, the account status is `active` at that host, and there is a functioning repository (with valid signature and commit `rev`), then the account has effectively been migrated.
+
+At a protocol level, this is how account migration works. And in some situations, when any previous PDS instance is unavailable or has inactive account status, getting in to the above state may be the only account recovery path available.
+
+However, in the common case, when there is an active account on a functional current PDS, a more seamless account migration process is possible. It can summarized in a few high-level steps:
+
+- creating an account on the new PDS (which may start with `active` false, such as `deactivated`)
+- migrating data from the old PDS to the new PDS
+- updating identity (DID document) to point to the new PDS and using a new atproto signing key. For PLC DIDs, usually involves updating PLC rotation keys as well.
+- updating account status on both PDS instances, such that old is inactive, and new is active
+- emitting a `#commit` event from the new PDS, signed by the new atproto signing key, with a higher commit `rev`.
```

Comment:
is there a dedicated/internal/"dummy" commit that needs to be emitted, or is this a "new skeet"/content commit? is a migration not valid/complete/official until that first commit emitted from the new PDS?

Reply:
I wonder if this is actually a requirement...? ie, if an account's DID doc is updated to point to a new PDS, its account is created there as active, its repo head there is the same `rev` as before, and its account is updated on the old PDS to be inactive, should other services still not recognize the migration if the repo is missing a new commit?

Reply:
This section is somewhat intentionally vague. We want "new PDS" to emit some kind of commit after the migration; it doesn't really matter if it is empty or not, so we don't mention or require that. empty/no-op is easy and what we would imagine most implementations would do.

the motivation for sending a commit like this is to have all downstream services try to validate the commit signature from this PDS, with a new rev. in theory this is redundant: if they got the identity event their caches are already updated, and if everything was smooth the next commit will come through fine. in practice, there are lots of things that might be slow on the first such commit, or may not have made the leap. we think sending a first commit event immediately after migration makes it more likely that actual initial interactions (post, like, whatever) will be successfully processed.

there is not practical way to require this: implementations might just not do it, and things would "probably" be fine? but it is valuable enough to mention and encourage here.